examples/msbr/openmc_msbr_model.py

```diff
@@ -0,0 +1,1045 @@
+plot.origin=(0, 0, 420)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-zoneIIA-upper2')
+plot.origin=(0, 0, 437)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-zoneIA-upper2')
+plot.origin=(0, 0, 439)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-zoneIIA-upper3')
+plot.origin=(0, 0, 440)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-zoneIA-upper3')
+plot.origin=(0, 0, 448)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-zoneIIA-upper4')
+plot.origin=(0, 0, 442)
+plot.width=(522.232, 522.232)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xy'
+plots.append(plot)
+
+plot = openmc.Plot(name='core-xz-detail-upper')
+plot.origin=(215, 0, 440)
+plot.width=(100, 100)
+plot.pixels=detail_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xz'
+plots.append(plot)
+
+plot = openmc.Plot(name='full-core-xz')
+plot.origin=(0, 0, 306.07)
+plot.width = (686.816, 612.14)
+plot.pixels=full_pixels
+plot.color_by='material'
+plot.colors=colormap
+plot.basis='xz'
+plots.append(plot)
+
+if plots_3d:
+    plot = openmc.Plot(name='full-core-3d')
+    plot.origin=(0,0,220)
+    plot.type = 'voxel'
+    plot.color_by='materials'
+    plot.colors=colormap
+    plot.width = (700., 700., 650.)
+    plot.pixels = (10000, 10000, 10000)
+
+    plots.append(plot)
+
+plots.export_to_xml()
+
+if deplete:
+    # Serpent fission q values
+    fiss_q = {
+        "Am243": 212952778.98135212,
+        "Cm246": 220179493.9541502,
+        "U239": 196182721.4907133,
+        "Np239": 198519429.0542043,
+        "Th233": 185840570.73897627,
+        "Cf251": 223166396.6983342,
+        "Am242": 215146730.16368726,
+        "Cm245": 214624022.18345505,
+        "Cf254": 230600814.3619568,
+        "Am241": 211216798.63643932,
+        "Th232": 197108389.42449385,
+        "Cm240": 219583368.40646642,
+        "Th231": 186918512.14598972,
+        "Bk246": 224446497.874413,
+        "Cm247": 218956599.9139631,
+        "U238": 206851381.70909396,
+        "Bk250": 225432928.78068554,
+        "U230": 198841127.68309468,
+        "Cf249": 221434495.10716867,
+        "U234": 200632850.9958874,
+        "Cm250": 219425761.1783332,
+        "Th229": 192235847.44789958,
+        "Cm241": 219075406.6897822,
+        "Pu237": 210593272.23024797,
+        "Am240": 215272544.02927735,
```

Comment:
Can you get these values from `pyne` or something?

Reply:
Nope. These are Q values calculated to match the fission q values used in Serpent. See [this script](https://github.com/openmc-dev/data/blob/master/depletion/generate_serpent_fissq.py) for more details.

Reply:
If the json file gets dumped somewhere -- can't you read it in? Or, if you have to run a serpent simulation once to obtain these values, can't you run it -- save the json file as read only -- and then read it in? It just seems wrong to me to have naked values like this.

Reply:
Just reiterating that I don't love having raw numbers like this. Ideally it would read something like
```py
if deplete:
  fiss_q = read_values()
  ....
  # or 
  with open("data.pkl", "rb") as serp_data:
    fiss_q = serp_data.load()
```
It just seems unsustainable to me.

Reply:
I agree with you. I will make it so a JSON file gets read in.